File: 0753/753.go
func crackingTheSafe(root *TreeNode) int {
	if root == nil {
		return 0
	}
	l := crackingTheSafe(root.Left)
	r := crackingTheSafe(root.Right)
	if l > r {
		return 1 + l
	}
	return 1 + r
}

File: 0753/753.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var crackingTheSafe = function(root) {
    if (!root) return 0;
    return 1 + Math.max(crackingTheSafe(root.left), crackingTheSafe(root.right));
};